app/helpers/number_helper.rb

```diff
@@ -69,8 +69,6 @@ def mhz_to_human_size(size, *args)
     nil
   end
 
-  private
-
```

Comment:
I'll second that.  I know that the private isn't doing anything for the class methods, but it is a visual queue that everything below it is not part of the public API for that class.

However, I avoid using `public` and would like to see that one :fire: